### PR TITLE
Issue 73

### LINKED ISSUE #73: Use ClinGen Allele Registry in CIViC Transform
Use ClinGen Allele Registry API with caid=`variant['allele_registry_id']` in CIViC transformation to get MANE transcript

### PR DESCRIPTION
**Pull request by korikuzma**
Closes #73

### CODE REVIEW

**Comment by korikuzma** [comment]
New commits:
- Merged new transform output and updated DB
    - `transform` now returns the index dictionary
- Clean up CLI

File: metakb/transform/civic.py [hunk 1]
```diff
@@ -616,44 +675,32 @@ def _get_therapy_descriptors(self, drug):
...
         for query in [ncit_id, label]:
             if not query:
                 continue
 
             therapy_norm_resp = self.therapy_query_handler.search_groups(query)
-            if therapy_norm_resp['match_type'] != 0:
-                break
+            if therapy_norm_resp['match_type'] > highest_match:
+                highest_match = therapy_norm_resp['match_type']
+                normalized_therapy_id = therapy_norm_resp['value_object_descriptor']['value']['therapy_id']  # noqa: E501
```
Comment (MayLiu27) [inline metakb/transform/civic.py:688]: not sure if you plan to do it in other branch but `therapy_id` should be changed to `id` as the normalizer was updated

**Review by MayLiu27 — APPROVED** [review]
Looks good. Just the `therapy_id` part see below

**Review by jsstevenson — APPROVED** [review]
It might be helpful to get the latest versions of gene/variant/disease up on PyPI today and update requirements here so that we can replicate the 'therapy_id' -> 'id' change for the other normalizers as well

**Comment by korikuzma** [comment]
@MayLiu27 @jsstevenson  Personally, I don't think the `{type}_id` -> `id` change is a huge deal to incorporate at the moment since it doesn't affect the response (currently only referencing descriptors by id) and think it should be resolved in a separate issue.

**Pull request merged**